### ISSUE TITLE
x.showbg() not implemented

### ISSUE THREAD

**Issue by chaosphere2112** [UV-CDAT/vcs#71]
we used to be able to plot in bg mode and at the end bring the canvas to the front.

``` python
import vcs
x=vcs.init()
x.plot([1,2,3,4,5,4,3,2,1],bg=1)  # plots in bg
x.showbg()  # would bring up the canvas
```

Migrated from: https://github.com/UV-CDAT/uvcdat/issues/1598

**Comment by doutriaux1** [comment]
@scottwittenburg would it be as easy as creating a new renwin and stick on it all the actors? Or is it more involved?

**Comment by scottwittenburg** [comment]
My first guess is it's probably more involved, but then, you know me: always looking on the gloomy side of things ;-)

I would have to spend some time looking a little deeper to know for sure.  And since there are some questions about how the bg mode is working anyway, we'll probably have to see about that first.  

Maybe @danlipsa would have a better idea, but I wonder if just setting the onscreen rendering property on the render window before doing another render would do it?

As a side note, I guess we should start sharing a new vocabulary around "context items", as we don't have "actors" anymore now that we have moved to Context2D.

**Comment by doutriaux1** [comment]
@scottwittenburg yes you're right `context items`. Let's not worry about it for now
